doc/api/console.md

```diff
@@ -77,9 +78,29 @@ const { Console } = require('console');
 const { Console } = console;
 ```
 
-### new Console(stdout[, stderr])
-* `stdout` {stream.Writable}
-* `stderr` {stream.Writable}
+### new Console(options)
+### new Console(stdout[, stderr][, ignoreErrors])
```

Comment:
@vsemozhetbyt The doctool doesn’t seem to be able to handle this anymore, since 9a6dd07e8d93c?

Reply:
Do you mean two consecutive heading? I cannot say why it should not. We have many of them in the `bufer.md` and they seem to be handled properly. Do you encounter issues with tests or doc building?

Reply:
Oh, I've tested this. It seems the consecutive heading we already have contains similar parameters and they do not differ from the list parameters below. Before 9a6dd07, these differences were checked, but [they did not throw, just reported to stderr](https://github.com/nodejs/node/blob/237cbe10fb82ba6ec69b14193a4a970e184306e7/tools/doc/json.js#L365-L370). It seems we should either split the section or change the parser strictness, which may not be good for detecting differences between signatures and parameter lists.

Reply:
BTW, consecutive headings are parsed (and were parsed) as clones: the section bodies are copied and checked identically: see [here](https://github.com/nodejs/node/blob/237cbe10fb82ba6ec69b14193a4a970e184306e7/tools/doc/json.js#L71-L81) and [here](https://github.com/nodejs/node/blob/237cbe10fb82ba6ec69b14193a4a970e184306e7/tools/doc/json.js#L482-L489). This is not changed in 9a6dd07, I've just make the differences throw.

Reply:
Compare How we usually split sections with different signatures:

https://nodejs.org/api/assert.html#assert_assert_fail_message
https://nodejs.org/api/assert.html#assert_assert_fail_actual_expected_message_operator_stackstartfunction

https://nodejs.org/api/buffer.html#buffer_class_method_buffer_from_array
https://nodejs.org/api/buffer.html#buffer_class_method_buffer_from_arraybuffer_byteoffset_lengt

https://nodejs.org/api/fs.html#fs_fs_write_fd_buffer_offset_length_position_callback
https://nodejs.org/api/fs.html#fs_fs_write_fd_string_position_encoding_callback

But with similar signatures:

https://nodejs.org/api/buffer.html#buffer_buf_readdoublebe_offset_noassert
https://nodejs.org/api/buffer.html#buffer_buf_readdoublele_offset_noassert

Reply:
So two heading with a similar name but different signatures do not confuse the JSON parser as far as the second signature is in sync with the parameter list below. However, it seems this is still the only case in our docs for now. And it can be a bit confusing: the first signature parameters are not openly listed (they are similar to object properties/options fields, but this is a bit non-formal way). Maybe we can split them, add an own minimal parameter list to the first signature and a link to the common description in the second section? It is a vague thing for me and it is not a strong opinion, so feel free to ignore) We still have a valid JSON (just a bit overflooded with parameter list clones) and the doc is still pretty readable in this way.